testing/trino-testing-kafka/src/main/java/io/trino/testing/kafka/TestingKafka.java

```diff
@@ -180,19 +189,21 @@ public void createTopicWithConfig(int partitions, int replication, String topic,
 
     public <K, V> RecordMetadata sendMessages(Stream<ProducerRecord<K, V>> recordStream, Map<String, String> extraProducerProperties)
     {
-        try (KafkaProducer<K, V> producer = createProducer(extraProducerProperties)) {
-            Future<RecordMetadata> future = recordStream.map(record -> send(producer, record))
-                    .reduce((first, second) -> second)
-                    .orElse(Futures.immediateFuture(null));
-            producer.flush();
-            return future.get();
-        }
-        catch (InterruptedException e) {
-            Thread.currentThread().interrupt();
-            throw new RuntimeException(e);
-        }
-        catch (ExecutionException e) {
-            throw new RuntimeException(e);
+        try (ThreadContextClassLoader ignored = new ThreadContextClassLoader(classLoader)) {
```

Comment:
during test execution from maven there is probably just one classloader, so current TCCL = classLoader